soc/arm/nxp_imx/rt6xx/soc.h

```diff
@@ -75,4 +75,21 @@
 /*!<@brief Slow mode */
 #define IOPCTL_PIO_SLEW_RATE_SLOW 0x80u
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if CONFIG_DISK_DRIVER_SDMMC &&					\
+	(DT_NODE_HAS_STATUS(DT_NODELABEL(usdhc1), okay) ||	\
+	 DT_NODE_HAS_STATUS(DT_NODELABEL(usdhc1), okay))
```

Comment:
This should be `usdhc2`

Reply:
Thank you, I have fixed this. I noticed the same error exists in the rt implementation. I have added a separate commit to fix this as well.